src/main/java/me/moodcat/backend/rooms/RoomInstance.java

```diff
@@ -273,10 +276,24 @@ public ChatMessageModel sendMessage(final ChatMessageModel model,
         return model;
     }
 
+    private void verifyNonSpamming(final User user) {
+        final long currentTime = System.currentTimeMillis();
```

Comment:
I was doubting if that was needed and @JWGmeligMeyling said it wasn't needed and shoud be deleted.
Therefore I deleted that part.